hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -608,9 +607,10 @@ static void xpadneo_update_battery(struct xpadneo_devdata *xdata, u8 value)
 		break;
 	default:
 		xdata->psy_desc.type = POWER_SUPPLY_TYPE_UNKNOWN;
+		break;
 	}
 
-	if (XPADNEO_BATTERY_CHARGING(value) || XPADNEO_BATTERY_PRESENT(value))
+	if (XPADNEO_BATTERY_CHARGING(value) || XPADNEO_BATTERY_PRESENT(value)) {
```

Comment:
Same here as above, I think the kernel formatting script added these again after I removed them. This probably also serves as a counter-measure against easy pitfalls. Also, technically speaking, the switch case statement isn't really a single statement. It's more like combined goto and if on steroids. I would argue differently if it wouldn't implicitly fall through.